.editorconfig

```diff
@@ -14,6 +14,9 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
 
+[*.json]
+indent_size = 4
```

Comment:
While it doesn't hurt anything, I don't think this should be needed. It's not in https://core.trac.wordpress.org/browser/trunk/.editorconfig - yet that manages to use 4-space tab for all file types anyway?

Reply:
Somehow the IDE didn't autoformat correctly if I didn't add this. However, now that's correctly formatted I guess we can revert this changes.